content/legal/release-policy.md

```diff
@@ -79,6 +79,11 @@ earlier, reporting can be done in the project's next scheduled board report.
 
 Projects SHALL publish official releases and SHALL NOT publish unreleased
 materials outside the development community.
+Projects SHALL provide one or more Project Download Pages as part of their web site.
+Projects with multiple sub-projects MAY provide multiple Project Download Pages
+that correspond to sub-projects.
```

Comment:
I don't think this sentence is needed, given that the surrounding statements make it clear that there can be more than one project download page.

Reply:
What is a sub-project? AIUI, PMCs have may have multiple projects. They do not have sub-projects.

If a PMC starts with one project, and then adds another, do they both become sub-projects? I think the wording is unnecessary and confusing.

Reply:
I propose to change it to "multiple artifacts" - that leaves enough freedom and does not introduce any new definition.